modules/rte_mis_core/src/Form/OverrideTermForm.php

```diff
@@ -49,6 +49,10 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       }
 
     }
+    elseif ($bundle == 'location') {
+      $form['langcode']['widget']['#description'] = $this->t('Please select the language for which you want to add location.');
+      $form['relations']['weight']['#access'] = FALSE;
```

Comment:
This should be applicable to all users except for app_admin.

Reply:
ok